variables.tf

```
@@ -276,6 +276,18 @@ variable "webdeploy_publish_basic_authentication_enabled" {
   nullable    = false
 }
 
+variable "storage_account_access_key" {
+  description = "The access key which will be used to access the backend storage account for the Function App. Conflicts with storage_uses_managed_identity."
+  type        = string
+  default     = null
+}
+
+variable "storage_uses_managed_identity" {
+  description = "Should the Function App use Managed Identity to access the storage account. Conflicts with storage_account_access_key."
+  type        = bool
+  default     = true
```

Update description and explicitly set `nullable = false`:

```suggestion
  description = "Should this Function App use its system-assigned identity to access the connected Storage account?"
  type        = bool
  default     = true
  nullable    = false
```